templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style type="text/css">
+        {# Specific override to allow larger width on this container#}
+        .container {
+            width: 1500px;
+        }
+        .bottom-pad {
+            padding-bottom: 25px;
+        }
+    </style>
+    <title>{{ "Forms Administration"|translate }}</title>
+</head>
+<body>
+<div class="container">
+    <form action="./forms_admin.php" method="post">
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="page-header">
+                <h1>{{ "Forms Administration"|translate }}</h1>
+            </div>
+        </div>
+    </div>
+    <div class="row bottom-pad">
+        <div class="col-xs-12">
+            <button type="submit" name="update" class="btn btn-default btn-save">{{ "Update"|translate }}</button>
+        </div>
+    </div>
+    {% if error %}
+        <div class="alert alert-danger">
+            {{ error|translate }}
```

Comment:
it's better to translate this upstream via xl()
(then it gets captured in the perl script that collects the new constants)